pkg/agent/config/traffic_encap_mode_test.go

```diff
@@ -1,3 +1,17 @@
+// Copyright ./hack/netpol/pkg/utils/utils_test.go Antrea Authors.
```

Comment:
why the Copy right start has strings `./hack/netpol/pkg/utils/utils_test.go`?

Reply:
Oh sorry, the wrong commit, have push the right now